packages/react-components/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@fluentui/react-components",
+  "version": "9.0.0-alpha.0",
+  "description": "Suite package for converged React components",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
```

Comment:
I think this package ought to have a webpack config so it will build a bundle that's usable in codepen. You can copy the one from react-cards (just change the `output` name).

Reply:
We currently don't have a clear agreement on whether we need UMD packages for converged.